pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -190,7 +190,7 @@ func (a *bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 	}
 	etcdEndpoints := make([]string, installConfig.MasterCount())
 	for i := range etcdEndpoints {
-		etcdEndpoints[i] = fmt.Sprintf("https://%s-etcd-%d.%s:2379", installConfig.Name, i, installConfig.BaseDomain)
+		etcdEndpoints[i] = fmt.Sprintf("https://%s-etcd-%d.%s:2379", installConfig.ObjectMeta.Name, i, installConfig.BaseDomain)
```

Comment:
> Stop relying on field promotion [1], because I'm about to add
InstallConfig.Platform.Name().

No sure how they coorelates. I think we can keep is as is.
When in future when we split installconfig to installconfig and clusterconfig; we can change to clustername

Reply:
> > Stop relying on field promotion [1], because I'm about to add InstallConfig.Platform.Name().
>
> No sure how they coorelates. I think we can keep is as is.

```console
$ git describe --dirty
v0.1.0-11-ga0a1dbc-dirty
$ git diff
diff --git a/pkg/asset/ignition/bootstrap/bootstrap.go b/pkg/asset/ignition/bootstrap/bootstrap.go
index a5226aa..51e6fc9 100644
--- a/pkg/asset/ignition/bootstrap/bootstrap.go
+++ b/pkg/asset/ignition/bootstrap/bootstrap.go
@@ -190,7 +190,7 @@ func (a *bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 	}
 	etcdEndpoints := make([]string, installConfig.MasterCount())
 	for i := range etcdEndpoints {
-		etcdEndpoints[i] = fmt.Sprintf("https://%s-etcd-%d.%s:2379", installConfig.ObjectMeta.Name, i, installConfig.BaseDomain)
+		etcdEndpoints[i] = fmt.Sprintf("https://%s-etcd-%d.%s:2379", installConfig.Name, i, installConfig.BaseDomain)
 	}
 
 	releaseImage := defaultReleaseImage
$ hack/build.sh 
++ dirname hack/build.sh
+ cd hack/..
+ MODE=release
++ git describe --always --abbrev=0 --dirty
+ LDFLAGS=' -X main.version=v0.1.0-dirty'
+ TAGS=
+ export CGO_ENABLED=0
+ CGO_ENABLED=0
+ case "${MODE}" in
+ TAGS=' release'
++ go env GOPATH
+ GOPATH=/home/trking/.local/lib/go/src/github.com/openshift/installer/vendor:/home/trking/.local/lib/go
+ go generate ./data
writing assets_vfsdata.go
+ echo ' release'
+ grep -q libvirt_destroy
+ go build -ldflags ' -X main.version=v0.1.0-dirty' -tags ' release' -o ./bin/openshift-install ./cmd/openshift-install
# github.com/openshift/installer/pkg/asset/ignition/bootstrap
pkg/asset/ignition/bootstrap/bootstrap.go:193:77: ambiguous selector installConfig.Name
```

So this is not compiling (with `ambiguous selector installConfig.Name`) unless I make this change.

Reply:
Sounds good.